teams/teams-ps/teams/Update-CsTeamTemplate.md

```diff
@@ -650,145 +649,152 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## NOTES
 
+ALIASES
+
 COMPLEX PARAMETER PROPERTIES
 
-To create the parameters described below, construct a hash table containing the appropriate properties.\

```

Comment:
I think I have originally misformatted the complex parameter properties. I have not manually tweaked them, it's just the visual changes